src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -74,38 +73,13 @@ object Config {
   object Blockchain {
     private val blockchainConfig = config.getConfig("blockchain")
 
-    val genesisDifficulty: Long = blockchainConfig.getLong("genesis-difficulty")
-    val genesisHash = ByteString(Hex.decode(blockchainConfig.getString("genesis-hash")))
```

Comment:
Should the `genesis-difficulty` and `genesis-hash` be removed from `reference.conf` as they are no longer used?